app/views/citizens/property_values/show.html.erb

```diff
@@ -12,13 +12,7 @@
         ) do %>
 
       <%= govuk_fieldset_header t('.field_set_header'), padding_below: 6 %>
-      <%= form.label :property_value, t('.label'), class: 'govuk-label' %>
-      <%= govuk_hint t('.hint'), id: :property_value_hint %>
```

Comment:
Can we remove `'.hint'` and `'.label'` from the locale file?
I think they are on lines  319 and 320